client/src/components/UserProfile.jsx

```diff
@@ -0,0 +1,187 @@
+import TextInput from './TextInput';
+import styled from 'styled-components';
+import { useEffect, useState } from 'react';
+import profilePic from '../images/ProfilePic.jpg';
+import Button from './Button';
+import DateInput from './DateInput';
+import dayjs from 'dayjs';
+const URL = process.env.REACT_APP_URL;
```

Comment:
I would prefer a blank line between your `imports` and your `consts` for better visibility of the latter. :)

Reply:
done :-)